README.md

```diff
@@ -103,6 +103,10 @@ When in SSH:
     cd /vagrant
     python3 chaos.py
 
+## Nbody simulation status
+
+![So chaotic](data/imgs/nbody.png)
```

Comment:
@RedPointyJackson currently you're getting the `nbody.png` from GitHub. That version doesn't change. You need to host the image on port 8080 or something, and then link to that.